test/unit/store/toast-store.js

```diff
@@ -127,4 +127,13 @@ describe('ToastStore', () => {
         assert.match(ToastStore.getToasts()[3].text, /Created a new Car Radio/);
     });
 
+    it('should add a toast message when creating a new item', () => {
+        Dispatcher.handleAction('ITEM_CREATED', {
+            address: 'iGwEZUvfA',
+            modelName: 'Resistor'
+        });
+        assert.lengthOf(ToastStore.getToasts(), 4);
+        assert.match(ToastStore.getToasts()[3].text, /Created a new Resistor/);
```

Comment:
Will do, but I'm not including the `!` because I don't think missing punctuation should fail a unit test like this.